src/mono/mono/mini/mini-arm.h

```diff
@@ -390,6 +390,12 @@ typedef struct MonoCompileArch {
 
 #define MONO_CONTEXT_SET_LLVM_EXC_REG(ctx, exc) do { (ctx)->regs [0] = (gsize)exc; } while (0)
 
+#if defined(HOST_WIN32)
```

Comment:
Guess this is needed just to get things building for HOST_WIN32? MONO_INIT_CONTEXT_FROM_FUNC is run on target host, so the code generated by below will never be executed when running cross compiler on Windows x64 host.

Reply:
Yes, this is to appease the compiler. MONO_INIT_CONTEXT_FROM_FUNC is used in a bunch of places in mini-exceptions.c, so I defined it as _something_

Reply:
A lot of things won't be executed when build into a cross compiler, some things are excluded using MONO_CROSS_COMPILE from build, other things will never be executed. As long it appease the compiler its good.